src/sumo_event.erl

```diff
@@ -53,7 +53,7 @@ dispatch(DocName, EventId, Event, Args) ->
     [] ->
       no_event_managers;
     EventManagers ->
-      lists:foreach(fun(EventManager) ->
+      _ = lists:foreach(fun(EventManager) ->
```

Comment:
you should use `ok` instead of `_` here. Actually, you don't even need the `_ = `. Not sure why dialyzer would complain about that (o_O)